capstone-server/src/main/java/com/hcl/capstoneserver/user/entities/Client.java

```diff
@@ -1,34 +1,23 @@
 package com.hcl.capstoneserver.user.entities;
 
-import com.hcl.capstoneserver.user.UserRole;
+import com.hcl.capstoneserver.user.UserType;
 
 import javax.persistence.Column;
 import javax.persistence.Entity;
 
 @Entity
-public class Client extends AppUser{
+public class Client extends Person {
     @Column(unique = true)
     private int clientId;
 
-    private String name;
-    private String address;
-    private String email;
-    private String phone;
-    private Float interestRate;
     private int accountNumber;
 
     public Client() {
     }
 
-    public Client(String userId, String password, int clientId, String name,
-                  String address, String email, String phone, Float interestRate, int accountNumber) {
-        super(userId, password, "CLIENT");
+    public Client(String userId, String password, UserType userType, String name, String address, String email, String phone, Float interestRate, int clientId, int accountNumber) {
```

Comment:
```suggestion
    public Client(String userId, String password, String name, String address, String email, String phone, Float interestRate, int clientId, int accountNumber) {
```

client userType is a constant. it should not be changed.